tests/foreman/ui/test_role.py

```diff
@@ -775,23 +812,493 @@ def test_positive_empty_filter_taxonomies_in_cloned_role(self):
+            5. Create an user group add above two users to the user group
+
+        :expectedresults:
+
+            1. Both the user should list role inherited from usergroup
+            2. User assigned to Organization B and Location B shouldn't have
+                access to the resources of organization A,B and Location A,B
+            3. User assigned to Organization A and Location A should have
+                access to the resources of organization A and Location A
+        """
+
+    @stubbed
+    @tier3
+    def test_negative_assign_org_admin_to_user_group(self):
+        """Users in usergroup can not have access to the resources in taxonomies if
+            the taxonomies of org admin role is not same
+
+        :id: 3b87f81a-2c9f-48e0-8bf1-863af335dd52
+
+        :steps:
+
+            1. Create an org admin role by cloning 'Organization admin' role
+            2. Assign an organization A and Location A to the org admin role
+            3. Create two users without assigning roles while creating them
+            4. Assign Organization B and Location B to both users
+            5. Create an user group add above two users to the user group
+
+        :expectedresults:
+
+            1. Both the user should list role inherited from usergroup
+            2. Both the user shouldn't have access to the resources of
+                organization A,B and Location A,B
+        """
+
+    @stubbed
+    @tier2
+    def test_negative_assign_taxonomies_by_org_admin(self):
+        """Org Admin doesnt have permissions to assign org/loc to any of
+        its entity
+
+        :id: 61a99863-801a-4d66-963b-9699ff1a3236
+
+        :steps:
+
+            1. Create Org Admin by cloning 'Organization admin' role
+            2. Assign an organization A,B and Location A,B to the org admin
+                role
+            3. Create user and assign above org admin role
+            4. Assign Organization A,B and Location A,B to the user
+            5. Login from the new user and attempt to assign organization(s)
+                and location(s) to any resource
+
+        :expectedresults: No Organization and Location tabs are available in
+            resource for Org Admin to assign the taxonomies
+        """
+
+    @stubbed
+    @tier1
+    def test_positive_remove_org_admin_role(self):
+        """Super Admin user can remove Org Admin role
+
+        :id: 9ef6e960-0775-4b1a-a17b-5bcbc4a865a9
+
+        :steps:
+
+            1. Create Org Admin by cloning 'Organization admin' role
+            2. Assign any taxonomies to it
+            3. Create an user and assign above role to user
+            4. Delete the org admin role
+
+        :expectedresults: Super Admin should be able to remove org admin role
+        """
+
+    @stubbed
+    @tier2
+    def test_positive_taxonomies_control_to_superAdmin_with_org_admin(self):
+        """Super Admin can access entities in taxonomies assigned to Org Admin
+        in org admin's presence
```

Comment:
Presence in the sence, Super Admin can access the entities in Org Admin taxonomies even though the Org Admin present(exists/not_deleted). 
In short what I want to test is - > The control of taxonomies are not completely taken by Org Admin from Super Admin, Super Admin can still access what Org Admin can access
Simply removing the presence